src/Taxonomy_Command.php

```diff
@@ -148,12 +148,22 @@ protected function get_counts( $taxonomies ) {
 	public function list_( $args, $assoc_args ) {
 		$formatter = $this->get_formatter( $assoc_args );
 
+		// Check if it's strict mode or not.
+		$is_strict = ( isset( $assoc_args['strict'] ) && true === $assoc_args['strict'] ) ? true : false;
+
+		unset( $assoc_args['strict'] );
+
 		if ( isset( $assoc_args['object_type'] ) ) {
 			$assoc_args['object_type'] = array( $assoc_args['object_type'] );
+			$taxonomy_object           = $assoc_args['object_type'];
+		} else {
+			$taxonomy_object = get_post_types();
 		}
 
 		$fields     = $formatter->fields;
-		$taxonomies = get_taxonomies( $assoc_args, 'objects' );
+		$taxonomies = ( $is_strict ) ?
```

Comment:
No need for the parentheses.
Also, when using ternaries in multiple lines, it is usually preferable to put the ternary operators at the beginning of the line.
```suggestion
		$taxonomies = $strict
```